tests/test_travis/check_ll_cirrus_lda.cpp

```diff
@@ -0,0 +1,188 @@
+// #include <unistd.h>
```

Comment:
Why are all these lines commented?

Reply:
This file is commented because it's an outdated version. The old test has now been removed and the travis test for lda can be found under the directory `tests/test_travis_lda`.